packages/config/src/config.js

```diff
@@ -167,7 +168,6 @@ module.exports = ({
             disableFallback
         }) : {
             contentBase: `${rootFolder || ''}/dist`,
-            hot: true,
```

Comment:
Can you leave it as an option?

Reply:
Have you found a case where this works and is useful? With a minimal config I've never seen this work with React components.